src/blog/announcing-tanstack-form-v2-alpha.md

```
@@ -0,0 +1,433 @@
+      run: schema,
+      triggers: ['change'],
+    },
+  ],
+})
+```
+
+<!-- ::end:tabs -->
+
+## Form Composition type safety
+
+#### Before (v1)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)